mobile-app/src/screens/navigationTypes.ts

```diff
@@ -0,0 +1,11 @@
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+
+export type RootStackParamList = {
+  Home: undefined;
+  Local: undefined;
+};
+
+export type HomeScreenProps = NativeStackScreenProps<RootStackParamList, 'Home'>;
+
+export type LocalScreenProps = NativeStackScreenProps<RootStackParamList, 'Home'>;

```

Comment:
Shouldn't it rather be like this ?

```suggestion
export type LocalScreenProps = NativeStackScreenProps<RootStackParamList, 'Local'>;
```

I'll merge it as is, since we need this PR and it seems to work anyway, but we should investigate 🕵️